src/main/kotlin/com/sainsburys/k2zpl/command/FieldData.kt

```diff
@@ -2,15 +2,33 @@ package com.sainsburys.k2zpl.command
 
 import com.sainsburys.k2zpl.builder.ZplBuilder
 
-internal data class FieldData(val data: String) : ZplCommand {
+internal data class FieldData(val data: String, val replaceNewlines: Boolean = false) : ZplCommand {
     override val command: CharSequence = "^FD"
     override val parameters: LinkedHashMap<CharSequence, Any?> = linkedMapOf("d" to data)
+
+    override fun build(stringBuilder: StringBuilder): StringBuilder {
+        return with(stringBuilder) {
```

Comment:
IIRC, there's no need for `with` because `StringBuilder.append` is a fluent chaining method, so we can replace it with:

```kotlin
stringBuilder
    .append(...)
    .append(...)
```

WDYT?

Reply:
Updated! 👍